ambry-messageformat/src/main/java/com.github.ambry.messageformat/DeleteMessageFormatInputStream.java

```diff
@@ -34,7 +34,7 @@
   public DeleteMessageFormatInputStream(StoreKey key, short accountId, short containerId, long deletionTimeMs)
       throws MessageFormatException {
     int headerSize = MessageFormatRecord.getHeaderSizeForVersion(MessageFormatRecord.headerVersionToUse);
-    int deleteRecordSize = MessageFormatRecord.Update_Format_V3.getRecordSize(UpdateRecord.Type.DELETE);
+    int deleteRecordSize = MessageFormatRecord.Update_Format_V3.getRecordSize(SubRecord.Type.DELETE);
```

Comment:
so `UndeleteMessageFormatInputStream.java` will be in the future PR?

Reply:
yeah that's not going to be in this one